ui/src/layouts/storage/functions.rs

```diff
@@ -0,0 +1,577 @@
+use std::{
+    ffi::OsStr,
+    path::{Path, PathBuf},
+    time::Duration,
+};
+
+use common::{
+    language::get_local_text,
+    state::{storage::Storage, Action, State},
+    warp_runner::{ConstellationCmd, FileTransferProgress, FileTransferStep, WarpCmd},
+    STATIC_ARGS, WARP_CMD_CH,
+};
+use dioxus_core::Scoped;
+use dioxus_desktop::DesktopContext;
+use dioxus_hooks::{
+    to_owned, use_coroutine, use_future, Coroutine, UnboundedReceiver, UseRef, UseSharedState,
+    UseState,
+};
+use futures::{channel::oneshot, StreamExt};
+use nix::sys::statvfs::statvfs;
```

Comment:
I cant recall but i believe this import is only compatible with unix-like systems. Could you confirm this works on windows too?

Reply:
Might want to look into `sysinfo` crate to see if that can add any help with this, which should work across platforms